app/com/m3/octoparts/repository/config/CacheGroupRepository.scala

```diff
@@ -33,12 +35,12 @@ object CacheGroupRepository extends ConfigMapper[CacheGroup] with TimestampsFeat
     many = HttpPartConfigRepository,
     throughFk = "cacheGroupId",
     manyFk = "httpPartConfigId",
-    merge = (group, partConfigs) => group.copy(httpPartConfigs = partConfigs)
+    merge = (group, partConfigs) => group.copy(httpPartConfigs = partConfigs.sorted)
```

Comment:
1. Do we really need to care about sorting at the repository level when it's primarily a UI concern? Seems inefficient.

2. If we must do sorting on this level, why are we not doing it using SQL ?